### PR TITLE
feature: bbgo completion

### PR DESCRIPTION
**Pull request by a-dwarf**
- Convenient use of the command line

### CODE REVIEW

**Comment by a-dwarf** [comment]
> 

0x56943DA2F80D20F01AfD91a4e98F32176051e9CF

File: doc/commands/bbgo_completion.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# bbgo Completion
```
Comment (c9s) [inline doc/commands/bbgo_completion.md:1]: Cool!!

Is this hand-written or automatically generated doc?

if it's hand-written, please move it to doc/topics, thanks!

Comment (a-dwarf) [reply doc/commands/bbgo_completion.md:1]: it's hand-written after I make bbgo.

Okey! I will move it to doc/topics.

Comment (a-dwarf) [reply doc/commands/bbgo_completion.md]: I have moved it to doc/topics.

**Comment by c9s** [comment]
the go.sum file needs to be updated.

**Review by c9s — APPROVED** [review]

**Comment by c9s** [comment]
thanks!

**Pull request merged**